cmds/core/uptime/uptime.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2013-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Get the time the machine has been up
+//
+// Synopsis:
+//     uptime
+//
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+	"time"
+)
+
+// This function takes in the filepath to a file containing loadavgs,it then extracts and returns the three load averages
+func loadavg(filepath string) (loadaverage string, err error) {
+
+	loadavg := strings.Fields(filepath)
+	loadavgstring := "loadaverage :" + loadavg[0] + "," + loadavg[1] + "," + loadavg[2]
+	return loadavgstring, nil
+}
+
+// This function takes in the filepath to a file containing the uptime,it then extracts and returns the uptime in the formart
+// Days , Hours , Minutes ,Seconds
+func uptime(filepath string) (uptime string, err error) {
+	uptimeArray := strings.Fields(filepath)
+	uptimeDuration, err := time.ParseDuration(string(uptimeArray[0]) + "s")
+	if err != nil {
+		return "-1", errors.New("Could not get uptime:Error parsing the string uptime to a time duration")
+	}
+	uptimeTime := time.Time{}.Add(uptimeDuration)
+	uptimeString := fmt.Sprintf("up %d days, %d hours, %d minutes ,%d seconds ,", uptimeTime.Day()-1, uptimeTime.Hour(), uptimeTime.Minute(), uptimeTime.Second())
+	return uptimeString, nil
```

Comment:
Functions should not return strings like this. 

The uptime function should probably return time.Time

The caller should figure out how to format

Reply:
if this was in a package I'd agree, but this is in a command as a helper function and there's a nice property here where the two functions can have the same signature.